container-search/src/main/java/com/yahoo/search/dispatch/SearchPath.java

```diff
@@ -149,7 +160,7 @@ private static SearchPath parseElement(String element) {
     }
 
     // an asterisk or an empty string followed by a comma or the end of the string
```

Comment:
Comment no longer correct.

Reply:
Fixed, it was not correct to start with.